device-types/Ubiquiti/UACC-PoE-plus-plus-10G.yaml

```diff
@@ -0,0 +1,72 @@
+---
+manufacturer: Ubiquiti
+model: 10G PoE++ Adapter (60W)
+part_number: UACC-PoE-plus-plus-10G
+slug: ubiquiti-uacc-poe-plus-plus-10g
+u_height: 0
+is_full_depth: false
+airflow: passive
+weight: 210
+weight_unit: g
+interfaces:
+  - name: Port 1
+    type: 10gbase-t
+    mgmt_only: false
+  - name: Port 2
+    type: 10gbase-t
+    poe_mode: pse
+    poe_type: type3-ieee802.3bt
+    mgmt_only: false
+power-ports:
+  - name: Power
+    type: iec-60320-c6
+    maximum_draw: 60
+front_image: true
+rear_image: true
+comments: |
```

Comment:
Thank you for your contribution,
Please reduce the amount of information in the comments property, this should only be a brief description with a link to the documentation.